packages/engine.io-client/lib/socket.ts

```diff
@@ -12,6 +12,7 @@ import {
   defaultBinaryType,
 } from "./globals.node.js";
 import debugModule from "debug"; // debug()
+import { nextTick } from "./globals.js";
```

Comment:
You can import the `nextTick` method from "./globals.node.js", so that it works in all environments.

Reply:
won't that make it only work in node? That file is hard coded to `process.nextTick`

https://github.com/socketio/socket.io/blob/fd99f2e15fef0e17ae9e3849c92ef0ca616a5919/packages/engine.io-client/lib/globals.node.ts#L1

Reply:
ah right